src/button/src/components/Button.js

```diff
@@ -3,31 +3,70 @@
  */
 
 import React, { Component } from 'react';
+import type { Node } from 'react';
 
-import StyledButton from './styled/StyledButton';
+import PropTypes from 'prop-types';
+
+import DefaultButton from './styled/DefaultButton';
+import PrimaryButton from './styled/PrimaryButton';
+import SecondaryButton from './styled/SecondaryButton';
+
+type ButtonType =
```

Comment:
I'm a concerned we're using the type prop to define the style as a regular HTML button already has a type attribute that is used for form submission and reset. It may cause confusion or unintended consequences for people who want to use this button component with the attribute.

<button type="submit">Click Me </button>

Reply:
Great feedback. I'll change this.